app/src/main/res/layout/activity_main.xml

```diff
@@ -1,16 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
+<RelativeLayout
     android:id="@+id/main_layout"
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
+    android:layout_height="wrap_content"
     android:orientation="vertical"
     tools:context=".MainActivity">
 
-    <com.sschuraytz.charcoaldrawing.DrawingView
-        android:id="@+id/canvas"
+    <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content" />
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
 
-</LinearLayout>
\ No newline at end of file
+        <com.sschuraytz.charcoaldrawing.DrawingView
```

Comment:
Both the DrawingView and the SeekBar should be able to go into the same RelativeLayout (main_layout) without having to be wrapped in a RelativeLayout/LinearLayout

Reply:
Good point. It involved tweaking some of the other layout settings for the SeekBar, but it's definitely cleaner code.